*** latex2html.bak	Thu Jun 15 11:00:09 2000
--- latex2html	Thu Jun 15 13:12:28 2000
***************
*** 1387,1393 ****
      };
      if ($ROOTED) {$TEXINPUTS .= "$envkey$FIXEDDIR"}
      $TEXINPUTS = &absolutize_path($TEXINPUTS);
!     $ENV{'TEXINPUTS'} = join($envkey,".",@_,$TEXINPUTS,$ENV{'TEXINPUTS'});
  }
  
  # provided by Fred Drake
--- 1387,1393 ----
      };
      if ($ROOTED) {$TEXINPUTS .= "$envkey$FIXEDDIR"}
      $TEXINPUTS = &absolutize_path($TEXINPUTS);
!     $TEXINPUTS = join($envkey,".",@_,$TEXINPUTS,$ENV{'TEXINPUTS'});
  }
  
  # provided by Fred Drake
